pgen-lib/build.gradle

```diff
@@ -0,0 +1,24 @@
+plugins {
+	id 'dev.nokee.cpp-library'
+	id 'cpp-unit-test'
+}
+
+ repositories {
+     // Use Maven Central for resolving dependencies.
+     mavenCentral()
+}
+
+library {
+    targetMachines = [machines.macOS.x86_64, machines.linux.x86_64]
+    targetLinkages = [linkages.static]
+ }
+
+ dependencies {
+    // assumes boost 1.80.0 is installed in /usr/local/boost for C++ unit tests
```

Comment:
It would be great if C++ could get its act together and make this automatic.

Reply:
Yeah, at some point I'm going to have to figure out how to do that....